host/src/c_api/c_api_mc.cc

```diff
@@ -669,7 +669,7 @@ bool attest_remote_report(
   // Verify the remote report to ensure its authenticity.
   result = oe_verify_report(NULL, remote_report, remote_report_size, &parsed_report);
   if (result != OE_OK) {
-    std::cout << "oe_verify_report failed " << oe_result_str(result) << std::endl;
+    LOG(FATAL) << "Remote attestation failed. Remote report verification failed.";
     goto exit;
```

Comment:
The goto calls are redundant because LOG(FATAL) will throw an exception.

Reply:
Fixed